README.md

```diff
@@ -1,22 +1,34 @@
-[RU]by [S]imple [E]xception [N]otification
-====
+<p align="center">
+  <a href="http://moove-it.github.io/rusen/">
+    <img src="https://moove-it.github.io/rusen/images/logo.png" alt="Rusen" />
+  </a>
+</p>
+
+<p align="center">
+  <a href="https://badge.fury.io/rb/rusen">
+    <img src="https://badge.fury.io/rb/rusen.png" alt="Gem Version">
+  </a>
+  <a href="https://codeclimate.com/github/Moove-it/rusen">
+    <img src="https://codeclimate.com/github/Moove-it/rusen.png" alt="Code Climate">
+  </a>
+  <a href="https://travis-ci.org/moove-it/rusen">
+    <img src="https://secure.travis-ci.org/moove-it/rusen.png?branch=master" alt="Build Status">
+  </a>
+  <a href="https://coveralls.io/github/moove-it/rusen?branch=master">
+    <img src="https://coveralls.io/repos/moove-it/rusen/badge.svg?branch=master&service=github" alt="Coverage Status">
+  </a>
+  <a href="https://inch-ci.org/github/moove-it/rusen">
+    <img src="https://inch-ci.org/github/moove-it/rusen.svg?branch=master" alt="Documentation Coverage">
+  </a>
+  <a href="http://www.rubydoc.info/github/moove-it/rusen">
+    <img src="https://img.shields.io/badge/yard-docs-blue.svg" alt="Documentation">
+  </a>
+</p>
 
 The Ruby Simple Exception Notification (a.k.a Rusen) gem provides a simple way for logging and sending errors in any ruby application.
 
 The notification includes information about the current request, session, environment and also gives a backtrace of the exception.
```

Comment:
"Notifications include information about the current request, session, environment. They also provide a backtrace of the exception."